src/tools/dev/scripts/bv_support/bv_silo.sh

```diff
@@ -142,45 +135,45 @@ Index: src/fpzip/pcdecoder.inl
 --- src/fpzip/pcdecoder.inl	(revision 809)
 +++ src/fpzip/pcdecoder.inl	(working copy)
 @@ -19,7 +19,7 @@
- T PCdecoder<T, M, false>::decode(T pred, unsigned context)
- {
-   // map type T to unsigned integer type
--  typedef typename M::RANGE U;
-+  typedef typename M::FPZIP_Range_t U;
-   U p = map.forward(pred);
-   // entropy decode d = r - p
-   U r = p + rd->decode(rm[context]) - bias;
+ T PCdecoder<T, M, false>::decode(T pred, unsigned context)
+ {
+   // map type T to unsigned integer type
+-  typedef typename M::RANGE U;
++  typedef typename M::FPZIP_Range_t U;
+   U p = map.forward(pred);
+   // entropy decode d = r - p
+   U r = p + rd->decode(rm[context]) - bias;
```

Comment:
I don't know why github thinks I changed this and other parts of bv_silo.sh. Telling github to ignore whitespace changes makes this diff disappear. I did all the editing on toss3 so I'm not sure how line endings could have gotten mangled, if that is indeed the problem.

Reply:
Maybe that file has bad line endings in it to begin with? Do you see `ctrl-M` chars in it by any chance?

Reply:
Yes, it has `ctrl-M` chars on the highlighted lines. My text editor automatically converted them when I edited the file, hence the changes.